qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -214,9 +214,9 @@ def convert_snapshot(self, shift, instruction):
             dict: Dictionary of required parameters.
         """
         command_dict = {
-            'name': 'snapshot',
```

Comment:
This is a confusing aspect of the Qobj spec, but the `name` in the terra api is just a reference for display purposes. Within the Qobj transport layer, `name` references the type of command which in this case is `snapshot`. Consequently, this change should be reverted.  See [the spec for more information](https://arxiv.org/abs/1809.03452).